v5/core/base_service.go

```diff
@@ -626,21 +694,6 @@ func NewRetryableHTTPClient() *retryablehttp.Client {
 	return client
 }
 
-// getRetryableHTTPClient returns the "retryable" Client hidden inside the specified http.Client instance
```

Comment:
didn't need this function here any more, but the retries-related tests rely on it, so I moved it there.